### ISSUE TITLE
Gravity readout on Lab Screen?

### ISSUE THREAD

**Issue by Denz1994** [phetsims/masses-and-springs#146]
It was mentioned by @arouinfar that a readout for the value of gravity may be necessary for the lab screen to solve for the mass of the mystery mass. I made a prototype for this readout in master (seen on lab screen). Any suggestions for improvements on this readout?

It is worth noting that selecting "Planet X" in the combo box causes the gravity to be shown. A workaround for this in pendulum lab was to include a text message instead of a readout.

**Comment by Denz1994** [comment]
After toying with some alternative solutions, we could have Planet X be removed from the Lab screen altogether. If we need a gravity readout to determine the mass of the mystery mass, then having an unknown gravity seems counterproductive.

Counter-argument: A user can find the Planet X gravity on the lab screen by using the adjustable mass which is labeled.

Counter-counter-argument: True, but ideally, this can be accomplished in the other screens and having them do it again for the same Plant X seems redundant.

Thoughts @arouinfar or @ariel-phet?

**Comment by ariel-phet** [comment]
I believe the decision in design meeting was to try and match the look and function of how this readout is handled in Pendulum lab (which seemed like a good suggestion).

**Comment by arouinfar** [comment]
@Denz1994 as @ariel-phet said, the decision was to format the Gravity slider like the one in Pendulum Lab. On Lab, it will look something like this:
![image](https://user-images.githubusercontent.com/8419308/34544414-5aa34288-f0a3-11e7-82af-859cdf12f4fb.png)

And on the other screens, it would look like: 
![image](https://user-images.githubusercontent.com/8419308/34544423-66c63c32-f0a3-11e7-9c63-28d3961cd85f.png)

In all cases, the combo box is below the slider track.

**Comment by Denz1994** [comment]
@arouinfar Could you review the gravity slider's visuals and critique it. JO and I went over the pendulum lab's gravity slider and we came to the conclusion that making it common code would not be best. There isn't much to leverage between the two sims aside from some visuals options in the code. 

Full context in #184.

**Comment by arouinfar** [comment]
@Denz1994 I made a few general pixel polishing suggestions regarding font sizes and alignment in #205, so I won't repeat here.

I've noticed some shifting when switching to Planet X.

On Intro, the combobox gets closer to the bottom of the panel, but the panel itself doesn't appear to shift.
<img src="https://user-images.githubusercontent.com/8419308/35538132-fdfe5b82-0509-11e8-86f1-f9fc471db433.png" width=300> <img src="https://user-images.githubusercontent.com/8419308/35538112-f5e9db1a-0509-11e8-8da5-b0eef05b51e4.png" width=300>

On other screens, the combo box shifts down a bit, increasing the panel height.
<img src="https://user-images.githubusercontent.com/8419308/35538078-df0f085c-0509-11e8-8f11-830dce409cf1.png" width=300> <img src="https://user-images.githubusercontent.com/8419308/35538101-e97f8334-0509-11e8-9f6f-c82566a2f8ef.png" width=300>
